docs/userguides/accounts.md

```diff
@@ -234,6 +234,22 @@ message = encode_defunct(text="Hello Apes!")
 signature = account.sign_message(message)
 ```
 
+## Signed Message
+
+Get the signed message signature as well as message hash signed by the account
+
+```py
+from ape.types.signatures import recover_signer
+
+[...]
+signature = account.sign_message(message)
+
+# signature
+print(signature.encode_vrs())
+# message hash
+print(signature.message_hash)
```

Comment:
Was picturing something where you could recover the signer using only the signature object as part of the example:
```suggestion
# get original signer
print(recover_signer(...))
```

Reply:
I don't think we can use `message_hash` to do that off chain. We have to use `ecrecover`